app/routes/goal_routes.py

```diff
@@ -1 +1,68 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from .. import db
+from .route_utilities import validate_model, create_model, get_models_with_filters
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return create_model(Goal,request_body)
+
+@bp.post("/<goal_id>/tasks")
+def add_tasks_to_goal(goal_id):
+    goal = validate_model(Goal, goal_id)
+    request_body = request.get_json()
+    task_ids = request_body.get("task_ids", [])
+    goal.tasks = []
+
+    for task_id in task_ids:
+        task = validate_model(Task, task_id)
+        goal.tasks.append(task)
+
+    db.session.commit()
+
+    return {"id": goal.id, "task_ids": task_ids}, 200
```

Comment:
```suggestion
    return {"id": goal.id, "task_ids":  [task.id for task in goal.tasks]}
```

`task_ids` is a list of ids that we get from the client in the request body (see line 18). Rather than sending a response back to the client that echos the input `task_ids` we should fetch the task ids directly from the goal (since we appended the new tasks to the existing goal), in case there were any other tasks already associated with this goal. 

Also, the default status code sent back to the client is 200 so we don't need need to specify 200 here.

Reply:
Fixed as requested.